docs/devs/Use-JerryScript-Debugger.md

```diff
@@ -23,12 +23,26 @@ The file argument is ignored in this case, therefore doesn't need to be specifie
 thus, there's no need to restart the environment if the remote source is changed.
 **Important note**: Remote sources must be sent in correct order! IoT.js compiles them in the order they are received, so file(s) used with `require` should be sent first, and the file(s) using them after.
 
+#### Select transport and protocol layers
+
+There are two available extension-provided transport layers, websocket and rawpacket, and two procotol layers, tcp and serial. Each initializes the debugger and blocks until a client connects. If you want to specify the debugger transmission layer (default: websocket) or transmission protocol (default: tcp) over the communication you can do with the `--debug-channel [websocket|rawpacket]` and `--debug-protocol [tcp|serial]` options:
```

Comment:
The title is a bit confusing.  There are many terms here. (transport layer, protocol layer, transmission layer, and transmission protocol.) How about using only `channel` and `protocol`?

e.g) `Select transport and protocol layers` - > `Select Channel and Protocol`